index.md

```diff
@@ -3,21 +3,25 @@ layout: default
 title: SapMachine
 ---
 
-**SapMachine** is a free, open-source, non-commercial, cross-platform, and fully-functional production-grade version of the Open Java Development Kit (OpenJDK).
-Built and supported by SAP, it provides a robust and reliable Java environment.
+**SapMachine** is the free, multiplatform, production-ready, and [Java SE certified](https://github.com/SAP/SapMachine/wiki/Certification-and-Java-Compatibility) [OpenJDK](https://openjdk.org/) (Open Java Development Kit) distribution by [SAP](https://sap.com). It serves as the default JDK for SAP's countless applications and services.
 
 <img align="left" width="240" src="assets/images/logo_circular.svg" alt="Logo of SapMachine">
 
-Following the release cadence of the OpenJDK, a new feature release is shipped every six months, with one designated as a long-term support release every two years.
-Update releases for active versions are provided every quarter. So SapMachine offers long-term support, regular performance improvements and timely security updates.
+This Distribution supports all major Operation Systems.
+SapMachine comes with long-term support releases that include bug fixes and performance updates; you can learn more about support and maintenance in our [wiki](https://github.com/SAP/SapMachine/wiki/Maintenance-and-Support).
 
-For SAP, the default JDK is SapMachine. It is the engine of countless applications and services, in cloud deployments as well as standalone on-premise installations at both, SAP and its customers.
+Our goal is to keep SapMachine as close to OpenJDK as possible,
+only adding additional features when absolutely necessary; you can find a list of these
+in the [wiki](https://github.com/SAP/SapMachine/wiki/Differences-between-SapMachine-and-OpenJDK). One major difference is that we started including [async-profiler](https://github.com/jvm-profiling-tools/async-profiler) in our distribution.
 
-SapMachine is available on a variety of Operating System/CPU architecture combinations, and it is certified to adhere to the Java SE standard.
-It can serve as a drop-in replacement for any JDK in Java-based applications, ranging from small desktop applications to high-performance, large-scale server applications.
+Our team's many contributions to the OpenJDK and the ecosystem include the [PowerPC/AIX support](http://openjdk.java.net/projects/ppc-aix-port/),
+ [helpful NullPointerExceptions](https://openjdk.org/jeps/358), a [website for JFR events](https://sap.github.io/SapMachine/jfrevents/),
+ our own [IntelliJ Profiler plugin](https://plugins.jetbrains.com/plugin/20937-java-jfr-profiler).
 
```

Comment:
Do not mention the IntelliJ Profiler plugin here because it's Johannes Bechberger, not SapMachine 😉

Reply:
Ok, but we already showed it to all our stakeholders and started to actively recruit someone to work on this plugin. So it's a project of SapMachine.

Reply:
Also it was part of the original version of the homepage and is prominently presented on the main page of the [wiki](https://github.com/SAP/SapMachine/wiki). So I think it is a SapMachine project. I totally agree that we should move the Plugin from my personal GitHub account to SAP, but that is out of scope for this PR.

Reply:
Yes, it is a SapMachine team project in a sense that we endorse it and you did it and still do it while working in the team. However, I would not mention it here on the front page yet, as long as the plugin is not really owned by sapmachine or SAP in IntelliJ.

Reply:
I remove it then

Reply:
I hope it is ok to still mention https://sap.github.io/SapMachine/jfrevents, because it is used by multiple people in our team and it's published on the SapMachine site.